cns/service/main.go

```diff
@@ -200,15 +275,18 @@ func main() {
 	logLevel := acn.GetArg(acn.OptLogLevel).(int)
 	logTarget := acn.GetArg(acn.OptLogTarget).(int)
 	logDirectory := acn.GetArg(acn.OptLogLocation).(string)
-	ipamQueryUrl, _ := acn.GetArg(acn.OptIpamQueryUrl).(string)
-	ipamQueryInterval, _ := acn.GetArg(acn.OptIpamQueryInterval).(int)
+	ipamQueryUrl := acn.GetArg(acn.OptIpamQueryUrl).(string)
+	ipamQueryInterval := acn.GetArg(acn.OptIpamQueryInterval).(int)
 	startCNM := acn.GetArg(acn.OptStartAzureCNM).(bool)
 	vers := acn.GetArg(acn.OptVersion).(bool)
 	createDefaultExtNetworkType := acn.GetArg(acn.OptCreateDefaultExtNetworkType).(string)
 	telemetryEnabled := acn.GetArg(acn.OptTelemetry).(bool)
 	httpConnectionTimeout := acn.GetArg(acn.OptHttpConnectionTimeout).(int)
 	httpResponseHeaderTimeout := acn.GetArg(acn.OptHttpResponseHeaderTimeout).(int)
 	storeFileLocation := acn.GetArg(acn.OptStoreFileLocation).(string)
+	privateEndpoint := acn.GetArg(acn.OptPrivateEndpoint).(string)
+	infravnet := acn.GetArg(acn.OptInfrastructureNetworkID).(string)
+	nodeID := acn.GetArg(acn.OptNodeID).(string)
```

Comment:
why do we need this? we are going to get these from config anyway

Reply:
checked with jaeryn..we should not advise customer to use cmdline args..its just for internal testing and it should be removed if not needed